packit_service_fedmsg/utils.py

```diff
@@ -41,6 +41,10 @@ def specfile_changed(body: dict) -> bool:
     Returns:
         `True` if the specfile has changed, `False` otherwise.
     """
+    if nested_get(body, "commit", "agent") == "pagure":
+        # skip the check if handling push from a PR
+        return True
```

Comment:
Can we move this skip out of this function so the function name still makes sense?